gaphor/diagram/connectors.py

```diff
@@ -339,8 +339,10 @@ def disconnect(self, handle: Handle) -> None:
 
 
 class RelationshipConnect(UnaryRelationshipConnect):
-    """
-    """
+    """Base for relationship connections between unique elements."""
+
+    def connect_subject(self, handle: Handle) -> bool:
```

Comment:
Why do we need this function? Now if someone forgets to implement it, it will not break

Reply:
You are correct, I thought one of the linters was giving me an error that one of the methods of the ABC wasn't implemented. :+1: